rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -69,6 +69,9 @@
     private static String diskReadBufferSizeConf = "hadoop.cache.data.disk.read.buffer.size";
     public static String socketReadTimeOutConf = "hadoop.cache.network.socket.read.timeout";
     private static String diskMonitorIntervalConf = "hadoop.cache.disk.monitor.interval";
+    private static String enableMeticsConf = "hadoop.cache.data.metrics.enabled";
```

Comment:
Should the namespace be something in rubix ?

Reply:
There's also a typo here. `metics` -> `metrics`

Reply:
Let us try to start a trend of having complete PRs. Along with code, a PR should have tests (unit and other types if required) and doc. By this rule, 
- the configuration parameter
- list of metrics
- JMX access

should be documented as part of this feature. Let us use correct names going forward and document them. Then we wont increase the backlog. 
For documentation - configuration and metrics, start a page and create a simple list. We can build it out as we go.